pyannote/metrics/base.py

```diff
@@ -36,6 +37,51 @@
 from pyannote.metrics.types import Details, MetricComponents
 
 
+# TODO:
+# - fit/unfit = proper terminology?
+def clone(metric):
```

Comment:
Can we make it a method of `BaseMetric`?

Reply:
It needs access to `self`, doesn't it? So regular method.
Unless there is something I don't understand about the distinction between method and statcimethod...